pset-functions.js

```diff
@@ -208,6 +302,34 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+  const words = phrase.split(' ')
+  const wordCount = {}
+
+words.forEach((word) => {
+  if (wordCount[word] === undefined) {
```

Comment:
Definitely my favorite piece of code here; I love that you are creating a new key/value pair if it does not already exist in empty object; great use of if/else to parse through the string.

Overall, really awesome job here Clo; you nailed every problem and came up with really creative solutions. Keep up the great work!